clang/lib/Sema/SemaType.cpp

```diff
@@ -7623,6 +7623,7 @@ handleNonBlockingNonAllocatingTypeAttr(TypeProcessingState &TPState,
   FunctionEffectSet FX(EPI.FunctionEffects);
   FunctionEffectSet::Conflicts Errs;
   bool Success = FX.insert(NewEC, Errs);
+  (void)Success;
```

Comment:
From the previous branch - was generating a warning when assertions are disabled.

Reply:
A better way to do this would be `[[maybe_unused]]` on the declaration instead of a cast to void.